alerts/google-gce-instances/memory-utilization-too-high.v1.json

```diff
@@ -0,0 +1,30 @@
+{
+  "displayName": "Memory Utilization",
+  "userLabels": {},
+  "conditions": [
+    {
+      "displayName": "VM Instance - Memory utilization",
+      "conditionThreshold": {
+        "filter": "resource.type = \"gce_instance\" AND metric.type = \"agent.googleapis.com/memory/percent_used\" AND metric.labels.state = \"used\"",
+        "aggregations": [
+          {
+            "alignmentPeriod": "300s",
+            "crossSeriesReducer": "REDUCE_NONE",
+            "perSeriesAligner": "ALIGN_MEAN"
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": "0s",
+        "trigger": {
+          "count": 1
+        },
+        "thresholdValue": 100
```

Comment:
How did you pick a threshold of `100` here? I would have expected something more like 80% or 90%

Reply:
Per chat, reducing this threshold to 90%.